src/blocks/modal.css

```diff
@@ -1,178 +1,208 @@
   width: 40px;
   height: 40px;
   background-image: url("../images/close.icon.svg");
-  background-color: transparent;
   background-repeat: no-repeat;
   background-size: contain;
-  transition: opacity 0.1s linear;
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+  position: absolute;
+  bottom: calc(100% + 5px);
+  left: calc(100% + 5px);
 }
 
 .modal__close:hover {
-  opacity: 60%;
-  cursor: pointer;
+  opacity: 0.5;
 }
 
-.modal__heading {
-  margin: 0;
-  padding-bottom: 44px;
+.modal__form {
+  width: 358px;
+  font-family: "Inter", Arial, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 17px;
 }
 
-.modal__form {
-  display: flex;
-  flex-direction: column;
-  gap: 24px;
+.modal__input {
   border: none;
-  font-size: 14px;
-  line-height: 1.2;
+  border-bottom: 1px solid #a9a9a9;
+  width: 100%;
+  margin-top: 30px;
+  padding-bottom: 13px;
 }
 
-.modal__label {
-  position: relative;
+.modal__input::placeholder {
+  color: #a9a9a9;
 }
 
 .modal__error {
   font-size: 12px;
   line-height: 15px;
   color: #ff0000;
-  width: 358px;
-  margin: 0;
+  opacity: 0;
+  position: relative;
+  top: calc(100% + 5px);
+  left: 0;
   transition: opacity 0.3s;
 }
 
 .modal__error_visible {
   opacity: 1;
-  left: 0px;
-  top: 5px;
-  position: relative;
-}
-
-.modal__input {
-  padding-bottom: 13px;
-  border: none;
-  border-bottom: 1px solid;
-  border-bottom-color: rgba(0, 0, 0, 0.2);
-  width: 100%;
 }
-
 .modal__input_type_error {
+  width: 100%;
+  box-sizing: border-box;
+  border: none;
   border-bottom: 1px solid red;
 }
 
-.modal__button {
-  background-color: #000;
-  color: #fff;
-  padding: 14px 124px 18px 124px;
-  margin-top: 18px;
-  border: solid;
-  border-radius: 2px;
-  font-size: 18px;
-  line-height: 1.2;
-  transition: opacity 0.1s linear;
-}
-
-.modal__button:hover {
-  opacity: 80%;
-  cursor: pointer;
-}
-
-.modal__button_disabled:hover {
-  opacity: 0.2;
-  cursor: default;
+.modal__image {
+  max-width: 100%;
+  max-height: 100%;
+  object-fit: cover;
 }
 
-.modal__button_disabled {
-  opacity: 20%;
-  cursor: default;
-}
-
-.modal__close_preview {
-  width: 30px;
-  height: 30px;
+.modal__caption {
+  font-family: "Inter", Arial, sans-serif;
+  margin: 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 14.5px;
   position: absolute;
-  top: -30px;
-  right: -30px;
+  top: calc(100% + 10px);
+  left: 0;
+  color: #080808;
 }
 
-.preview-image {
-  max-width: 75vw;
-  max-height: 75vh;
+.modal__button {
+  background-color: black;
+  color: #fff;
+  width: 100%;
+  height: 50px;
+  margin-top: 48px;
+  margin-left: auto;
+  border: none;
+  border-radius: 2px;
+  cursor: pointer;
 }
 
-.modal__preview-text {
-  margin: 0;
-  margin-top: 8px;
+.modal__button:hover:not(:disabled) {
+  opacity: 0.8;
 }
 
-.modal__image-container {
-  position: relative;
+.modal__button:disabled {
+  /*color: #a9a9a900; */
+  color: #a9a9a9;
```

Comment:
Needs correction: Please use `background` instead of `color` here, so the button will have grey background and white text.